mail/dovecot/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dovecot
-PKG_VERSION:=2.3.5.2
+PKG_VERSION:=2.3.6
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.dovecot.org/releases/2.3
-PKG_HASH:=ba14e41aefd81a868a35b83bcb54194116106424d37690519b50ea83c0f31bf2
+PKG_HASH:=ed1d8dc1beeae9c6c73deac73a62ef19fe9262fbffd86604a3f690452f5536c7
 PKG_LICENSE:=LGPL-2.1 MIT BSD-3-Clause Unique
```

Comment:
Can you update the LGPL tag?
https://spdx.org/licenses/
What does Unique refer to?

Reply:
is has LGPL and MIT, I don't know either what is with the unique stuff

Reply:
all stuff is at the beginning of https://github.com/dovecot/core/blob/master/COPYING

Reply:
The LGPL "tag" isn't valid, please see the link I provided.
Drop the unique tag
It's not complete by better than before after those changes